dist/rules/no-internal-barrel-imports.js

```diff
@@ -116,19 +116,19 @@ const rule = {
     const { ignoredBarrelModules = [], requiredBarrelModules = [] } =
       typeof maybeTsConfig === "string" && extraOpts
         ? {
-            ignoredBarrelModules:
-              extraOpts[OPTION_IGNORED_BARREL_MODULES] &&
-              resolvePathsOption(
-                extraOpts[OPTION_IGNORED_BARREL_MODULES],
-                maybeTsConfig
-              ),
-            requiredBarrelModules:
-              extraOpts[OPTION_REQUIRED_BARREL_MODULES] &&
-              resolvePathsOption(
-                extraOpts[OPTION_REQUIRED_BARREL_MODULES],
-                maybeTsConfig
-              ),
-          }
+          ignoredBarrelModules:
```

Comment:
i dont mind too much, but why is there a ton of extra white space changes in this pr?

Reply:
Since I made some code changes I ran the VS Code formatter, it's the result of that